components/Popover/arrowProperties.js

```diff
@@ -0,0 +1,60 @@
+import Colors from '../Colors';
+
+const ARROW_SIZE = '16px';
+const SIDE_POSITION_X = -8;
+const CENTER_POSITION = '50%';
+
+const getColorBySkin = skin => {
+  const indexColor = skin.toUpperCase();
+  return skin === 'default' ? Colors.WHITE : Colors[indexColor][200];
+};
+
+const getArrow = (placement, skin) => {
+  let placementPosition = '';
+  const placementPositionTop = `
+  content: '▼';
+  left: ${CENTER_POSITION};
+  text-shadow: 0px 4px 4px ${Colors.SHADOW[50]};
+  bottom: 0;
+  transform: translateX(-${CENTER_POSITION});`;
+
+  switch (placement) {
```

Comment:
Why `switch` instead of an object? 

REF: https://github.com/catho/quantum/blob/master/components/Button/skins.js#L4

💅

Reply:
Done.